### DISCUSSION TITLE
Default PV inverter and grid meter

### DISCUSSION THREAD

**Discussion by Caseyin** [freakent/dbus-mqtt-devices#128] [Q&A] [answered]
I installed the software and when i reboot my cerbo i get default
My Grid meter
My PV Inverter
Both with the name SMA?
Why is this and how to remove it?
We have an SMA PV inverter running in the building, is this automatically detected??

![Screenshot from 2024-10-08 12-05-03](https://github.com/user-attachments/assets/0c07010e-8ab6-4906-81be-a4fa0bf2371b)

**Comment by freakent** [comment]
Hi,

What do you mean by SMA?

After you installed the driver what else did you do? Nothing will appear on that screen unless you register a device with the appropriate MQTT messages. 

Nothing gets auto detected, you have to publish a device registration message over MQTT. 

Click on one of the devices and then click on the Device menu. You should be able to see the details it was registered with.

**Reply by Caseyin** [reply]
When i check with dbus-spy i see this.
I haven' t done anything else but running your node-red demo with temperature and tank
Is it possible that our SMA inverter is auto detected by Victron? The main difference is now that i have enabled the MQTT which wasn't before
![Screenshot from 2024-10-08 12-33-56](https://github.com/user-attachments/assets/ac6be58b-7796-4599-a1a9-85efcdef000c)
![Screenshot from 2024-10-08 12-38-52](https://github.com/user-attachments/assets/73417781-1d89-4a1c-b022-6d9738210ed0)

**Reply by Caseyin** [reply]
![Screenshot from 2024-10-08 12-52-48](https://github.com/user-attachments/assets/ff1a2ab9-6518-4165-a02a-cb97dc12fe4b)

**Reply by freakent** [reply]
Well if you ran the node-red sample then that is exactly what you would expect. The node-red sample code demonstrates how to register devices. You should also find there is an inject node that you can use to send a disconnect message. 

Nothing gets auto detected by this driver. 

Please read the README about how the driver works and the registration protocol.

**Comment by Caseyin** [comment, marked as answer]
SOLVED
Eventually solved by reinstalling the Venus OS from scratch by SD-card. Apparently a Shelly python-script was the cause of this weird behavior. Now it work like a charm. Very nice and solid solution. Sorry for the inconvenience and your wasted time. I will press the thank you button :-)

**Reply by freakent** [reply]
I'm glad to hear you got things working. Thanks for the coffee!
